js/pillbox.js

```diff
@@ -298,48 +296,45 @@
 			}
 
 			if (items.length) {
-				$.each(items, function (i, item) {
+				var newItems = [];
+				$.each(items, function prepareItemForAdd (i, item) {
 					var $item = $(item.el);
-					var $neighbor;
 
 					$item.attr('data-value', item.value);
 					$item.find('span:first').html(item.text);
 
 					// DOM attributes
 					if (item.attr) {
-						$.each(item.attr, function (key, value) {
+						$.each(item.attr, function handleDOMAttributes (key, value) {
 							if (key === 'cssClass' || key === 'class') {
 								$item.addClass(value);
 							} else {
 								$item.attr(key, value);
 							}
 						});
-
 					}
 
 					if (item.data) {
 						$item.data('data', item.data);
 					}
 
-					$newHtml.push($item);
+					newItems.push($item);
```

Comment:
Well, it appears that this takes an array of DOM elements (items does not appear to be an array of jQuery elements, but, rather just normal DOM objects) and creates an array of jQuery elements. So, just saying `newItems = items` would not be equivalent.